setup.py

```diff
@@ -10,16 +11,20 @@ def description():
             lines.append(line)
         return "".join(lines).strip()
 
-setup(name='terminal-colors',
-	version='3.0.0',
-	description='Utility to test color capabilities of terminal.',
-	long_description=description(),
-	author='John Eikenberry',
-	author_email='jae@zhar.net',
+install_requires = [
+    "windows-curses ; platform_system=='Windows'",
+]
+
+setup(
+    name='terminal-colors',
+    version='3.0.3',
+    description='Utility to test color capabilities of terminal.',
+    long_description=description(),
+    author='John Eikenberry',
+    author_email='jae@zhar.net',
     url="https://github.com/eikenb/terminal-colors",
-    scripts=['terminal-colors'],
-    packages=find_packages(),
-	classifiers=[
+    packages=["terminal_colors"],
```

Comment:
could be changed back to `packages=find_packages()`, but that would require adding `__init__.py` as a sibling of `terminal_colors.py`